otter/integration/tests/test_convergence.py

```diff
@@ -508,6 +509,7 @@ def test_reaction_to_oob_server_deletion_below_min(self):
                 self.rcs, self.scaling_group, N_SERVERS / 2)(
                 self.scaling_group.trigger_convergence))
         )
+    test_reaction_to_oob_server_deletion_below_min.tags = ["CATC-004"]
```

Comment:
One aesthetic suggestion: make this into a decorator, so you can just say `@tagged("CATC-004")` before the test method.